Discord.Addons.Interactive/InlineReaction/InlineReactionCallback.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Discord.Commands;
+using Discord.WebSocket;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Discord.Addons.Interactive
+{
+    public class InlineReactionCallback : IReactionCallback
+    {
+        public RunMode RunMode => RunMode.Sync;
```

Comment:
Should this not be `RunMode.Async`?

Reply:
No idea to be honest, I used `PaginatedMessageCallback` as a template.